lib/collection/cookie.js

```diff
@@ -208,7 +208,7 @@ _.assign(Cookie.prototype, /** @lends Cookie.prototype */ {
              * Any extra parameters that are not strictly a part of the Cookie spec go here.
              * @type {Array}
              */
-            extensions: options.extensions || undefined
+            extensions: _.isArray(options.extensions) ? options.extensions : undefined
```

Comment:
Do we have a test that checks the behaviour when `extensions` is not an `array`?

Reply:
Fixed.